process/logsevents/logsAndEventsProcessor.go

```diff
@@ -51,74 +60,62 @@ func (lep *logsAndEventsProcessor) ExtractDataFromLogsAndPutInAltered(
 	logsAndEvents map[string]nodeData.LogHandler,
 	preparedResults *data.PreparedResults,
 	timestamp uint64,
-) (data.TokensHandler, tags.CountTags) {
-	txsMap := converters.ConvertTxsSliceIntoMap(preparedResults.Transactions)
-	scrsMap := converters.ConvertScrsSliceIntoMap(preparedResults.ScResults)
+) (data.TokensHandler, tags.CountTags, map[string]*data.ScDeployInfo) {
+	lep.txsMap = converters.ConvertTxsSliceIntoMap(preparedResults.Transactions)
```

Comment:
you changed the behavior of this struct from stateless to stateful. Now, I strongly suggest to extract a new struct that will hold all these elements along with the associated functions. It will prevent hard to reproduce and fix bugs that can appear in a new function that somehow alter these states.

Reply:
done.